### ISSUE TITLE
Autumnal Cleanup 2016 Reporting

### ISSUE THREAD

**Issue by d12frosted** [syl20bnr/spacemacs#7334]
Thanks everyone for participating in the event! All issues are already taken. After several days I will update stats! Stay tuned!

---

This issue is used to report contributors' work on the [**:fallen_leaf: Autumnal Cleanup 2016**](https://github.com/syl20bnr/spacemacs/wiki/Autumnal-Cleanup-2016).

If you want to take part of this collective effort, you can read a presentation [here](https://github.com/syl20bnr/spacemacs/wiki/Autumnal-Cleanup-2016) and discuss it on [Gitter](https://gitter.im/syl20bnr/spacemacs).

---

**Important!** In order to participate in this event you **must** ask for issues before reporting them. Any issues reported without being assigned to you are not counted. In order to be assigned, ask @d12frosted in on [Gitter](https://gitter.im/syl20bnr/spacemacs) or by [email](mailto:boris@d12frosted.io). For example,

> Could you please send me 10 random issues?

Or even be more specific.

> Could you please send me 20 issues related to Haskell, OCaml or Erlang?

---

**Please do not ask or discuss anything here.**

---

This page is only used for reporting progress as comments in the following format:

```
- [ ] #<ISSUE_NUMBER> <FLAG>
```

As for instance:

```
- [ ] #1 Valid
- [ ] #2 Wontfix
- [ ] #3 NRA
- [ ] #4 Can't reproduce
```

_The checkboxes will be used by repository maintainers to mark their progression on dealing with issues, so please do not check any issue yourself._

---

Some statistics about the :fallen_leaf: [Autumnal Cleanup 2016](https://github.com/syl20bnr/spacemacs/wiki/Autumnal-Cleanup-2016) progress:

Contributions:
- gilbertw1: 230 (27.09%)
- NJBS: 98 (11.54%)
- mahinshaw: 90 (10.60%)
- smile13241324: 75 (8.83%)
- deb0ch: 66 (7.77%)
- bleggett: 50 (5.89%)
- jredville: 50 (5.89%)
- lazywithclass: 50 (5.89%)
- rgrinberg: 30 (3.53%)
- kdelwat: 30 (3.53%)
- cmr: 25 (2.94%)
- bmag: 20 (2.36%)
- ksjogo: 10 (1.18%)
- ekmecic: 10 (1.18%)
- cro: 5 (0.59%)
- inirudebwoy: 5 (0.59%)
- mwillsey: 3 (0.35%)
- robbyoconnor: 1 (0.12%)
- alimoeeny: 1 (0.12%)

Overall progress: 849/900 (94.33%)

Go read the the description page if you want to be part of it :-)

**Comment by bmag** [comment]
- [ ] #4123 Valid
- [ ] #4136 Can't Reproduce (labeled as a feature request, but the feature already existed at the time and still exists, and actually sounds like a local bug that I can't reproduce)
- [ ] #6038 Valid
- [ ] #6322 Valid
- [ ] #6471 Valid
- [ ] #6645 Valid (upstream bug)
- [ ] #6683 Discussion Over
- [ ] #7060 Valid
- [ ] #7238 Valid
- [ ] #7302 Valid (upstream bug, might require work-around)

**Comment by NJBS** [comment]
- [ ] #4174 Valid
- [ ] #4176 Valid, however, conflicts must be resolved and the README's keybindings list shouldn't include function names.
- [ ] #4224 Valid
- [ ] #4297 Valid, however, conflicts must be resolved
- [ ] #4325 Valid, however, ido is no longer part of spacemacs-base so the changes will have to be moved to spacemacs-completion. Additionally, it looks like the author closed the issue associated with this PR by mistake, so the `PR pending` tag was automatically removed and should be re-added to https://github.com/syl20bnr/spacemacs/issues/4324
- [ ] #4339 Valid, however, the issue is now that `K` is bound to `org-agenda-capture` in the evilified state.
- [ ] #5023 Valid
- [ ] #5221 Wontfix
- [ ] #5364 Discussion Over, should maybe be reported upstream but the original question seems to be answered.
- [ ] #5677 No Response, I'm unable to reproduce this on current `develop`.
- [ ] #5792 Valid, should report upstream, flycheck does not see react-mode as a valid mode for eslint, https://github.com/flycheck/flycheck/blob/master/flycheck.el#L7657.
- [ ] #5801 NRA, fixed by Emacs 25.1.
- [ ] #5826 Valid
- [ ] #6520 Valid, should probably be reported upstream to clean-aindent-mode.
- [ ] #6579 Valid
- [ ] #7090 Unsure, I don't understand why those settings need to be in `user-init`, and they seem to work fine on my end when put in `user-config`.
- [ ] #7093 Valid
- [ ] #7119 Valid
- [ ] #7219 Valid
- [ ] #7298 Valid

**Comment by deb0ch** [comment]
- [ ] #3659 Discussion Over
- [ ] #5063 Valid
- [ ] #5088 Discussion Over
- [ ] #5166 Unsure (might have been fixed upstream but not sure)
- [ ] #5310 Valid, however, conflicts must be resolved and config variable documented
- [ ] #6531 Discussion Over, solution proposed in #6575 but not merged yet.
- [ ] #6592 Discussion Over
- [ ] #6844 Wontfix
- [ ] #6909 Discussion Over
- [ ] #7027 Valid

**Comment by robbyoconnor** [comment]
[ ] #4958 Ember.js -- I made this PR but it's been dormant

**Comment by jredville** [comment]
- [ ] #3965 Valid
- [ ] #4262 Unsure
- [ ] #4956 Unsure
- [ ] #5870 Valid
- [ ] #5950 NRA
- [ ] #6033 Can't Reproduce
- [ ] #6377 Valid
- [ ] #6516 NRA
- [ ] #6887 Valid, but no response since Aug 23
- [ ] #7067 No Response

**Comment by deb0ch** [comment]
[ ] #6347
[ ] #6441
[ ] #
[ ] #
[ ] #
[ ] #
[ ] #
[ ] #

**Comment by deb0ch** [comment]
[ ] #6347 Wontfix
[ ] #6441Valid
[ ] #6528 Discussion over (although I still personnally find that it is an issue)
[ ] #6567 Unsure (don't know if fixed upstream, but if not wont be fixed and workaround needed on spacemacs side. Ping me if necessary.)
[ ] #7113 Valid
[ ] #7300 Valid
[ ] #7342 Valid
[ ] #7357 Valid

**Comment by deb0ch** [comment]
- [ ] #6347 Wontfix
- [ ] #6441Valid
- [ ] #6528 Discussion over (although I still personnally find that it is an issue)
- [ ] #6567 Unsure (don't know if fixed upstream, but if not wont be fixed and workaround needed on spacemacs side. Ping me if necessary.)
- [ ] #7113 Valid
- [ ] #7300 Valid
- [ ] #7342 Valid
- [ ] #7357 Valid

**Comment by cro** [comment]
- [ ] #4938 Cannot reproduce (using 0.200.1@25.1.1) 
- [ ] #5947 Discussion over
- [ ] #6396 Discussion over
- [ ] #6431 Wontfix
- [ ] #6938 Fixed

**Comment by deb0ch** [comment]
- [ ] #3687 Can't reproduce
- [ ] #3695 Discussion Over (Solutions were proposed but no PRs made)
- [ ] #3722 No Response
- [ ] #4649 Valid (mergeable, but no response since January)
- [ ] #5375 Discussion Over
- [ ] #5911 No Response (Bug is still here, I experience it everyday, but no solution proposed and no activity. Should we close the issue and open a new one on the same topic for it to get a "refresh" ?)
- [ ] #5965 Discussion Over (the idea is still very good, just waiting for a PR)
- [ ] #6200 Discussion Over 
- [ ] #6932 Unsure (should report upstream, don't know if it was reported or fixed (no github repo))
- [ ] #7135 Unsure (Reported upstream but not fixed yet. No solutions proposed upstream.)

**Comment by mahinshaw** [comment]
- [ ] #3762 Can't Reproduce - Closed by initiator
- [ ] #3809 Valid - However, the discussion inf #4562 might be relevent to this issue.
- [ ] #4562 Valid
- [ ] #5186 Valid
- [ ] #5199 Valid - This is a problem upstream in evil mode.
- [ ] #5304 Fixed
- [ ] #5634 Valid
- [ ] #5777 Can't Reproduce
- [ ] #6504 No Response - It's a feature request with takers around implementation.
- [ ] #6878 Valid - PR pending #7183

**Comment by NJBS** [comment]
- [ ] #4207 Unsure, I'm unable to reproduce the issue where inputting a capital letter causes `helm-projectile-find-file` to slow down. The other issue seems to be an issue with the nvm so I'm unsure what spacemacs can do about this.
- [ ] #4481 Fixed in Develop, that particular FAQ entry has been directly moved into the OSX section of the main documentation. https://github.com/syl20bnr/spacemacs/#macos
- [ ] #4587 Discussion Over
- [ ] #4970 Discussion Over, the proposed solution is claimed to be buggy (I haven't tested it for myself).
- [ ] #5035 NRA, `C-x C-f` no longer uses ido.
- [ ] #5738 Valid
- [ ] #5800 Unsure, not sure if 25.1 or 0.201 fixes these issues.
- [ ] #5889 Not Reproducible, I'm unable to reproduce this issue on the latest `develop`.
- [ ] #6179 Valid, cider-repl-mode derives from fundamental-mode.
- [ ] #6262 Valid
- [ ] #6353 Valid
- [ ] #6387 Valid, looks like its been in the merging process since July.
- [ ] #6412 Valid, however, conflicts must be fixed.
- [ ] #6751 Valid, insider builds of bash on windows might fix this so hopefully this can be resolved when they reach the main release channel.
- [ ] #6944 Not reproducible
- [ ] #6982 No Response
- [ ] #6993 Valid
- [ ] #7053 Valid
- [ ] #7190 Valid
- [ ] #7266 Fixed, should be closed by https://github.com/syl20bnr/spacemacs/commit/bbe29b9cc17eee0255add896e7e98cab15f3f5dc

**Comment by mahinshaw** [comment]
- [ ] #5459 Can't Reproduce
- [ ] #5667 Valid - This is a request for keybindings, without an investment to implement those keybindings.
- [ ] #7068 NRA - Deprecated var in cider refer to [Changelog](https://github.com/clojure-emacs/cider/blob/de7d734dd2551522eeac48c65c581c0b1e1c325f/CHANGELOG.md#L550). Can be closed.
- [ ] #7209 Won't Fix - The conversation is around a package not provided by the clojure layer or Spacemacs in general.
- [ ] #7282 Fixed in Develop
- [ ] #4504 No Response
- [ ] #5427 No Response - The concept here seems out of the scope of spacemacs in general
- [ ] #5437 Fixed
- [ ] #7066 Valid
- [ ] #7076 Valid

**Comment by lazywithclass** [comment]
- [ ] #3822 Can’t Reproduce, worked fine for me, last comment on 6 Dec 2015
- [ ] #4205 Can’t Reproduce, No Response, https://github.com/syl20bnr/spacemacs/pull/468
- [ ] #4290 Valid
- [ ] #4591 Discussion Over
- [ ] #4618 Valid
- [ ] #4713 No Response
- [ ] #4940 Valid, reproduced and opened an issue in magit
- [ ] #5648 Valid, fix has been suggested, awaiting PR
- [ ] #6750 Wontfix
- [ ] #6942 Unsure

**Comment by rgrinberg** [comment]
```
- [ ] #5897 No Response. Although I just pinged the author.
- [ ] #6080 Wontfix
- [ ] #7167 Unsure. Seems like the author has a tentative fix, but it's not good enough to be merged.
- [ ] #7235 Can't Reproduce. Worked fine for me but recent issue.
- [ ] #5158 Valid
- [ ] #6093 No Response
- [ ] #6521 Not Relevant Anymore.
- [ ] #6703 Valid
- [ ] #7268 Valid
- [ ] #7312 Valid
```

**Comment by ksjogo** [comment]
Starting work.
- [ ] #4477 Not verified
- [ ] #4892 Not verified
- [ ] #5202 Not verified
- [ ] #5323 Not verified
- [ ] #5449 Not verified
- [ ] #6075 Not verified
- [ ] #6301 Not verified
- [ ] #6505 Not verified
- [ ] #6630 Not verified
- [ ] #7070 Not verified

**Comment by gilbertw1** [comment]
- [ ] #3742 Valid
- [ ] #4127 Valid - Should be closed in favor of #6583 which addresses underlying issue
- [ ] #4837 Valid - reported upstream
- [ ] #5057 Wontfix - graphics related question, no recent activity
- [ ] #5114 Valid
- [ ] #5243 No Response - feature request, no response / activity
- [ ] #5498 Valid - outstanding pull request against vi-tilde-fringe
- [ ] #6005 Discussion Over - answered question
- [ ] #6190 Discussion Over - answered question / platform specific workaround provided
- [ ] #6385 Can't Reproduce - worked fine for me in current version w/ similar environment

**Comment by deb0ch** [comment]
- [ ] #6389 Valid
- [ ] #6466 Discussion Over
- [ ] #6864 Valid
- [ ] #6969 Fixed
- [ ] #7123 Unsure
- [ ] #7249 Fixed
- [ ] #7322 Valid

**Comment by gilbertw1** [comment]
- [ ] #4115 Not verified - Couldn't verify no OSX install available
- [ ] #4355 Disussion Over - No consensus, looks like members abandoned discussion
- [ ] #4524 Not Relevant Anymore - reporter resolved issue, comments consist of similar but unrelated issues / workarounds
- [ ] #4851 Fixed
- [ ] #4924 Fixed
- [ ] #5491 Can't Reproduce - tested using steps in StreakyCobra's comment and could not reproduce
- [ ] #6421 Valid - Reproduced issue in master and develop w/ Emacs 25.x
- [ ] #6610 Fixed in Develop
- [ ] #6999 Can't Reproduce
- [ ] #7328 Already Closed - Fixed

**Comment by gilbertw1** [comment]
- [ ] #3678 No Response - pr w/ conflicts over a year old, no activity or consensus on feature request
- [ ] #4337 No Response - no response / activity on feature request; nearly a year old
- [ ] #4602 Fixed - packages are listed when updating packages in current spacemacs build
- [ ] #4778 Discussion Over
- [ ] #5296 Discussion Over - question w/ satisfactory answer
- [ ] #5379 Discussion Over
- [ ] #5416 Valid
- [ ] #5476 Not Sure - PR with no activity since March
- [ ] #5559 No Activity - PR appears to have been rejected with no followup in a while
- [ ] #5924 Valid
- [ ] #6065 Discussion Over
- [ ] #6085 Valid
- [ ] #6300 Valid
- [ ] #6443 No Response
- [ ] #6661 Valid
- [ ] #6717 Valid
- [ ] #6743 Discussion Over - question w/ satisfactory answer
- [ ] #7018 Valid
- [ ] #7170 Valid
- [ ] #7234 Valid

**Comment by gilbertw1** [comment]
- [ ] #3716 Can't Reproduce
- [ ] #3880 No Response
- [ ] #4060 Discussion Over
- [ ] #4338 Unsure - no response in 10+ months, setup too complex to test easily
- [ ] #4619 Valid - appears to be an emacs problem
- [ ] #4807 Discussion Over - question turned forum w/ no recent activity
- [ ] #5144 Valid
- [ ] #5426 Valid
- [ ] #5594 Discussion Over - Question answered
- [ ] #5910 Valid - although associated PR seems to be abandoned
- [ ] #6394 Valid
- [ ] #6416 Valid - reproduced in master & develop
- [ ] #6628 Valid - should be closed, already tracked by #2872
- [ ] #6699 Valid
- [ ] #6947 Valid
- [ ] #7086 Valid
- [ ] #7088 Already Closed
- [ ] #7160 Valid
- [ ] #7179 Valid
- [ ] #7192 Valid

**Comment by mahinshaw** [comment]
- [ ] #3988 Valid
- [ ] #4366 Valid
- [ ] #4936 Valid
- [ ] #5424 Valid
- [ ] #5614 Valid
- [ ] #6391 Valid
- [ ] #6632 Valid
- [ ] #6792 Discussion Over
- [ ] #7159 Valid
- [ ] #7325 Can't Reproduce

**Comment by mwillsey** [comment]
- [ ] #5320 Valid
- [ ] #5327 Valid
- [ ] #6324 Valid

**Comment by smile13241324** [comment]
- [ ] #5294 Already closed
- [ ] #6015 Already closed
- [ ] #6537 Discussion Over -  Issue is caused by general behaviour of outline, no spacemacs issue
- [ ] #6810 Already closed
- [ ] #6831 Already closed

**Comment by gilbertw1** [comment]
- [ ] #3895 Discussion Over - question answered
- [ ] #3996 Unsure - discussion appears to be over
- [ ] #4020 Fixed - ack search displays DONE when finished in helm
- [ ] #4032 Fixed
- [ ] #4250 Valid
- [ ] #4706 No Response
- [ ] #4950 No Response
- [ ] #5037 Discussion Over
- [ ] #5092 No Response
- [ ] #5255 Discussion Over
- [ ] #5429 Valid
- [ ] #5574 Valid
- [ ] #6369 Valid - should be closed in favor of #6387
- [ ] #6535 Can't Reproduce
- [ ] #6648 Valid
- [ ] #6715 Valid
- [ ] #6936 Valid
- [ ] #7189 Valid
- [ ] #7236 Valid
- [ ] #7314 Valid

**Comment by smile13241324** [comment]
- [ ] #4403 Already closed
- [ ] #4685 Already closed
- [ ] #5396 Already closed
- [ ] #5544 Already closed
- [ ] #5783 Already closed
- [ ] #6117 Already closed
- [ ] #6524 Valid
- [ ] #6763 Already closed
- [ ] #6830 Valid
- [ ] #7214 Already closed

**Comment by bleggett** [comment]
- [ ] #3689 Wontfix (local cygwin environment issue, works fine with recommended Windows builds)
- [ ] #4000 Can't Reproduce (no issue with latest Org/spacemacs 200.2 release on Windows) 
- [ ] #6197 Can't Reproduce (Works fine for me with custom font+daemon mode on Windows)
- [ ] #3848 Discussion Over (resolved/abandoned a year ago)
- [ ] #5266 No Response (PRs abandoned some time ago)
- [ ] #5687 Wontfix (not a Spacemacs issue)
- [ ] #6296 Can't Reproduce (no repro with latest org/spacemacs 200.2/emacs25)
- [ ] #6499 Valid
- [ ] #7233 Unsure (no 4k display to test with)
- [ ] #7253 Valid

**Comment by smile13241324** [comment]
- [ ] #6578 Already closed
- [ ] #6828 Valid
- [ ] #6925 Already closed
- [ ] #6992 Already closed
- [ ] #7015 Already closed
- [ ] #7050 Already closed
- [ ] #7071 Already closed
- [ ] #7168 Already closed
- [ ] #7169 Already closed
- [ ] #7206 Discussion Over - Every layer is responsible for it's own README.md file.

**Comment by smile13241324** [comment]
- [ ] #4231 Already closed
- [ ] #4272 Already closed
- [ ] #5190 Already closed
- [ ] #5256 Already closed
- [ ] #5299 Valid
- [ ] #5431 Already closed
- [ ] #5471 Already closed
- [ ] #5991 Valid - New layer only supports auto-complete, code change necessary to also allow company prior to merge.
- [ ] #6023 Already closed
- [ ] #6034 Already closed
- [ ] #6261 Already closed
- [ ] #6365 Already closed
- [ ] #6436 Wontfix - Issue is caused by cygwin not providing proper unicode support. Linked ticket is closed, so this ticket should be closed too.
- [ ] #6449 Already closed
- [ ] #6460 Already closed
- [ ] #6608 Already closed
- [ ] #6707 Can't Reproduce - Checked on develop and master branch on linux and windows. Using emacs 25.1.
- [ ] #6774 Valid
- [ ] #6836 Valid - Still valid as #6828 is also valid. PR needs to be merged.
- [ ] #6994 Already closed
- [ ] #7017 Already closed
- [ ] #7020 Already closed
- [ ] #7055 Already closed
- [ ] #7315 Already closed
- [ ] #7317 Already closed

**Comment by cmr** [comment]
- [ ] #4175 Not verified
- [ ] #4189 Not verified
- [ ] #4977 Not verified
- [ ] #4980 Not verified
- [ ] #5019 Not verified
- [ ] #5061 Not verified
- [ ] #5100 Not verified
- [ ] #5208 Not verified
- [ ] #5419 Not verified
- [ ] #5438 Not verified
- [ ] #5470 Not verified
- [ ] #5533 Not verified
- [ ] #5636 Not verified
- [ ] #5996 Not verified
- [ ] #6563 Not verified
- [ ] #6708 Not verified
- [ ] #6747 Not verified
- [ ] #6879 Not verified
- [ ] #6894 Not verified
- [ ] #6972 Not verified
- [ ] #7006 Not verified
- [ ] #7009 Not verified
- [ ] #7101 Not verified
- [ ] #7289 Not verified
- [ ] #7330 Not verified

**Comment by mahinshaw** [comment]
- [ ] #3735 Valid
- [ ] #3943 Valid
- [ ] #4063 No Response
- [ ] #4348 No Response
- [ ] #4705 No Response
- [ ] #4752 No Response
- [ ] #5140 Discussion Over - Should be closed, as a solution is provided to opener.
- [ ] #5395 Valid - Upstream issue.
- [ ] #5596 No Response
- [ ] #5818 Valid
- [ ] #5915 No Response - Implementer/Opener is no longer working on the PR.
- [ ] #6012 No Response
- [ ] #6146 Can't Reproduce
- [ ] #6228 Valid
- [ ] #6328 Valid
- [ ] #6595 Valid
- [ ] #7035 No Response
- [ ] #7182 Valid
- [ ] #7327 Can't Reproduce
- [ ] #7333 Can't Reproduce

**Comment by smile13241324** [comment]
- [ ] #3755 Already closed
- [ ] #3946 Already closed
- [ ] #4154 Already closed
- [ ] #4155 Already closed
- [ ] #4246 Already closed
- [ ] #4247 Already closed
- [ ] #4708 Already closed
- [ ] #5280 Already closed
- [ ] #5543 Already closed
- [ ] #5609 Already closed
- [ ] #6018 Already closed
- [ ] #6067 Already closed
- [ ] #6104 Already closed
- [ ] #6269 Already closed
- [ ] #6380 Already closed
- [ ] #6468 Already closed
- [ ] #6587 Discussion Over - New package, has been created in separate issue, this thread seems to be dead. Package issue #6604.
- [ ] #6621 Already closed
- [ ] #6631 Valid
- [ ] #6655 Already closed
- [ ] #6659 Already closed
- [ ] #6973 Valid
- [ ] #7026 Already closed
- [ ] #7041 Already closed
- [ ] #7186 Already closed

**Comment by bleggett** [comment]
- [ ] #4196 Can't Reproduce - Works with current Spacemacs/Emacs 25.
- [ ] #4490 Valid - Recommend merge PR and close.
- [ ] #4494 Valid
- [ ] #4718 Valid
- [ ] #5240 Valid
- [ ] #5474 Can't Reproduce
- [ ] #5606 Discussion Over
- [ ] #5718 Valid
- [ ] #5776 Fixed
- [ ] #5824 Valid
- [ ] #5992 Valid
- [ ] #6219 Can't Reproduce
- [ ] #6583 Valid
- [ ] #6618 Valid
- [ ] #6971 Valid
- [ ] #7112 Valid
- [ ] #7158 Discussion Over
- [ ] #7163 Valid
- [ ] #7223 Valid
- [ ] #7276 Fixed In Develop

**Comment by deb0ch** [comment]
- [ ] #3813 Unsure (old bug, no solutions or origin proposed, not sure if still present, can't reproduce)
- [ ] #3870 Already closed
- [ ] #4375 Already closed
- [ ] #4484 Valid
- [ ] #4666 Unsure (MacOS-specific, not sure if it is really an issue)
- [ ] #4862 Valid (Needs conflicts resolution. Yet another PR pending for a long time.)
- [ ] #4943 Valid (Reported upstream, no activity for a long time. Upstream issue still opened)
- [ ] #5708 Valid

**Comment by smile13241324** [comment]
#6578 Unsure - Not able to get scala working
#6828 Valid
#6925 Can’t Reproduce
#6992 Valid - Dicussion asks for different key bindings in transient states however no pr has been made.
#7015 Valid - PR waiting for merge
#7050 Wontfix
#7071 Fixed - Ticket was a question, solution was posted and is possible in master
#7168 Valid - PR is still waiting to be merged.
#7169 Valid
#7206 Discussion Over - Every layer is responsible for it's own README.md file.

**Comment by smile13241324** [comment]
#4403 Valid - Required nimsuggest binary is not mentioned in layer Readme.md
#4685 Unsure - Tested on Linux worked as expected, however this should be reproduced on osx instead.
#5396 Wontfix - User has problem with his own system, no spacemacs issue.
#5544 Valid
#5783 Unsure - Seems I am not able to get clojure to run on my pc
#6117 Valid
#6524 Valid
#6763 Valid
#6830 Valid
#7214 Already closed

**Comment by smile13241324** [comment]
- [ ] #5294 Wontfix - No spacemacs issue but misunderstanding on user side.
- [ ] #6015 Cannot reproduce
- [ ] #6537 Discussion Over - Issue is caused by general behaviour of outline, no spacemacs issue
- [ ] #6810 Cannot reproduce - Tried on develop and master, modeline looks fine in spacemacs-dark theme and leuven.
- [ ] #6831 Fixed - User asked for way to configure certain keys, solution works

**Comment by deb0ch** [comment]
- [ ] #5841 Valid (even though not updated since April)
- [ ] #5856 No Response
- [ ] #6194 Can't reproduce
- [ ] #6450 No Response, Can't reproduce
- [ ] #6706 Discussion Over
- [ ] #6773 Valid
- [ ] #6900 Valid
- [ ] #6960 Can't reproduce
- [ ] #7104 Valid
- [ ] #7140 Already closed
- [ ] #7181 Unsure (apparently fixed, need confirmation from @robbyoconnor)
- [ ] #7237 Valid

**Comment by gilbertw1** [comment]
- [ ] #4746 Discussion Over
- [ ] #5187 No Response
- [ ] #5787 Valid
- [ ] #5857 Valid - Reproduced on master & develop
- [ ] #5891 No Response - associate PR closed
- [ ] #6323 Valid
- [ ] #6606 No Response
- [ ] #6638 No Response
- [ ] #6668 Discussion Over - question answered
- [ ] #6966 Valid
- [ ] #6980 Can't Reproduce
- [ ] #6996 Valid
- [ ] #7001 Valid
- [ ] #7008 Valid
- [ ] #7052 Valid
- [ ] #7063 Already Closed
- [ ] #7103 Valid
- [ ] #7153 Already Merged
- [ ] #7208 Valid
- [ ] #7243 Valid

**Comment by NJBS** [comment]
- [ ] #3930 No Response, not an OSX user so sadly I cannot answer this one.
- [ ] #3945 Valid, should probably be tagged as ready for work and beginner friendly.
- [ ] #4062 Valid, I can still reproduce this on latest `develop`.
- [ ] #4230 No Response
- [ ] #4418 Valid, it seems people can still reproduce this although personally I cannot. (perhaps to be reproduced should be removed since it seems multiple users are able to reproduce the issue)
- [ ] #4905 Can't reproduce, wasn't able to reproduce this one on latest `develop`, although based on Andre's response it seems that this bug might be hard to reproduce.
- [ ] #4914 Valid
- [ ] #4971 No Response
- [ ] #5125 Valid, however, conflicts must be resolved.
- [ ] #5465 Valid
- [ ] #5663 Valid
- [ ] #5741 No Response, I'm also not able to reproduce this.
- [ ] #5956 No Response
- [ ] #6094 Valid, although it seems like there isn't any work being done on the `nlinum-relative` end to alleviate these issues.
- [ ] #6253 Valid
- [ ] #6461 No Response
- [ ] #6553 No Response
- [ ] #6670 Valid, should probably be beginner friendly and ready for work.
- [ ] #6911 Valid
- [ ] #7273 Not Sure, I think this might be a possible duplicate of #5007

**Comment by d12frosted** [comment]
It's just my marker. Ignore it 😸

**Comment by gilbertw1** [comment]
- [ ] #4018 Discussion Over - no activity on proposal in over a year
- [ ] #4307 Discussion Over
- [ ] #4411 Valid - looks valid, but no activity in a while
- [ ] #4478 Discussion Over
- [ ] #4531 No Response
- [ ] #4552 Can't Reproduce
- [ ] #4655 Valid - looks valid, but no activity in ~8 months
- [ ] #4915 Valid - looks valid, but no activity in ~7 months
- [ ] #4979 Discussion Over
- [ ] #5107 Can't Reproduce
- [ ] #5168 Valid - looks valid, pr has conflicts and hasn't been updated in a while
- [ ] #5287 Valid - reproduced against master & develop
- [ ] #5353 Discussion Over
- [ ] #5365 Valid
- [ ] #5452 Discussion Over
- [ ] #5607 Valid
- [ ] #5641 Discussion Over
- [ ] #5867 Can't reproduce
- [ ] #5875 Can't reproduce
- [ ] #6056 Valid - looks valid, no activity in ~5 months
- [ ] #6081 Can't reproduce
- [ ] #6144 Valid
- [ ] #6272 Valid
- [ ] #6440 Can't Reproduce
- [ ] #6456 Valid - can't reproduce, but seems to be affecting a number of users
- [ ] #6575 Valid
- [ ] #6652 No Response - no response in several months to feature request
- [ ] #6680 Can't Reproduce 
- [ ] #6788 Valid
- [ ] #6871 Discussion Over
- [ ] #6899 Valid
- [ ] #6965 Already Closed
- [ ] #7111 Valid
- [ ] #7150 Valid
- [ ] #7202 Valid
- [ ] #7229 Valid
- [ ] #7245 Already Closed
- [ ] #7255 Already Closed
- [ ] #7256 Already Closed
- [ ] #7280 Valid

**Comment by mahinshaw** [comment]
- [ ] #3861 Can't Reproduce
- [ ] #4104 Can't Reproduce
- [ ] #4486 No Response
- [ ] #4493 No Response
- [ ] #4533 Valid
- [ ] #4836 No Response
- [ ] #5117 No Response
- [ ] #5303 Can't Reproduce
- [ ] #5329 Can't Reproduce
- [ ] #5632 Valid
- [ ] #5682 No Response
- [ ] #6050 Valid
- [ ] #6124 Valid
- [ ] #6235 No Response
- [ ] #6299 Valid - Upstream problem
- [ ] #6533 Valid
- [ ] #6940 Valid
- [ ] #7022 Can't Reproduce
- [ ] #7092 Valid - However, Won't fix is a more likely stance to take with this one.
- [ ] #7210 Valid - Like the above, Won't fix would be nice.

**Comment by kdelwat** [comment]
- [ ] #3772 Fixed / Can't reproduce
- [ ] #3883 Valid
- [ ] #4031 No Response
- [ ] #4124 Discussion Over
- [ ] #4219 Valid
- [ ] #4952 Discussion Over	(not enough information given to reproduce)
- [ ] #5261 Valid
- [ ] #5907 Discussion Over (continued in #5910)
- [ ] #6242 Can't reproduce
- [ ] #6629 Wontfix (upstream/environment problem)

**Comment by lazywithclass** [comment]
- [ ] #4106 Valid
- [ ] #5059 Can't Reproduce
- [ ] #5276 Wontfix, Discussion Over
- [ ] #5343 Valid
- [ ] #5462 Valid
- [ ] #5521 Valid
- [ ] #6121 Valid
- [ ] #6772 Valid
- [ ] #7264 Valid
- [ ] #7320 Fixed
- [ ] #3911 Not Relevant Anymore
- [ ] #4356 Not verified
- [ ] #5232 No Response
- [ ] #5259 Valid
- [ ] #5487 No Response
- [ ] #6096 Discussion Over
- [ ] #6362 Not Relevant Anymore (reported upstream)
- [ ] #6410 Valid
- [ ] #6752 Valid
- [ ] #6796 No Response

**Comment by smile13241324** [comment]
- [ ] #4231 Valid
- [ ] #4272 Unsure - Tried it on linux and it works as expected. However complain was made explicity for emacs on osx therefore someone else with an apple should try that too.
- [ ] #5190 NRA - User complains about not working default keymap however the keys he is complaining about are not longer mapped to the given values, therefore this request should be closed.
- [ ] #5256 Can't Reproduce - Tried it with fish and zsh both worked well.
- [ ] #5299 Valid
- [ ] #5431 Valid
- [ ] #5471 NRA - User complained about not working feature, then realised he was using the wrong key binding think this is not longer applicable and should be closed.
- [ ] #5991 Valid - New layer only supports auto-complete, code change necessary to also allow company prior to merge.
- [ ] #6023 Unsure - Tested on linux on develop, no issues here. However as the issue has been reported on OSX I think someone from the apple users should try this too.
- [ ] #6034 Can't Reproduce - Tested on develop and master with the attached file. Worked as expected.
- [ ] #6261 Can't Reproduce - Tested on develop. When tree is moved via `SPC w L` the tree adjusts its size to the assigned window. However `g g` still works as expected now. When the tree is refreshed via `g r` the original size is restored. This should be closed.
- [ ] #6365 Valid - PR waiting to be merged
- [ ] #6436 Wontfix - Issue is caused by cygwin not providing proper unicode support. Linked ticket is closed, so this ticket should be closed too.
- [ ] #6449 Unsure - Well I have to admit I don't understand the ticket maybe someone with more experience in RUST can help here.
- [ ] #6460 Valid
- [ ] #6608 Can't Reproduce - Checked on develop and master branch on linux. Works as expected.
- [ ] #6707 Can't Reproduce - Checked on develop and master branch on linux and windows. Using emacs 25.1.
- [ ] #6774 Valid
- [ ] #6836 Valid - Still valid as #6828 is also valid. PR needs to be merged.
- [ ] #6994 Already closed
- [ ] #7017 Valid - When I set `(setq tab-always-indent t)` in my user-config and then open a new .js file the value is automatically set back to 'complete'.
- [ ] #7020 Fixed in Develop
- [ ] #7055 No Response - A new feature was asked for, however no one responded since over a month.
- [ ] #7315 Valid - Was able to reproduce on latest develop on linux.
- [ ] #7317 Already closed - User had a question which was answered has been closed by the user.

**Comment by gilbertw1** [comment]
- [ ] #3756 Can't Reproduce
- [ ] #3863 No Response - No response in almost a year on PR & failed CI check
- [ ] #3865 Can't Reproduce - also, no activity in almost a year
- [ ] #3887 Unsure - Don't have FreeBSD install, no activity in almost a year
- [ ] #4009 Valid - although PR has conflicts and no activity in ~8 months
- [ ] #4725 Valid - although PR has conflicts and no activity in ~8 months
- [ ] #4741 Discussion Over
- [ ] #4785 Valid - in merging state and appears valid, but no activity in several months
- [ ] #4815 Valid
- [ ] #5130 Already Closed
- [ ] #5237 Discussion Over
- [ ] #5239 No Response
- [ ] #5241 Discussion Over
- [ ] #5302 Discussion Over
- [ ] #5669 Discussion Over
- [ ] #5671 Valid
- [ ] #5806 Can't Reproduce
- [ ] #5862 Valid - appears valid, but no activity in several months
- [ ] #5979 Can't Reproduce
- [ ] #6166 No Response - reporter hasn't responded in several months
- [ ] #6203 Valid - successuflly reproduced against master & develop (adding evil-in-single-undo fixes issue)
- [ ] #6209 Valid
- [ ] #6425 Valid
- [ ] #6569 Can't Reproduce
- [ ] #6571 Valid
- [ ] #6616 Can't Reproduce
- [ ] #6654 Valid
- [ ] #6690 Fixed - PR w/ documentation changes merged
- [ ] #6693 Valid
- [ ] #6722 Valid
- [ ] #6918 Valid
- [ ] #7059 Already Closed
- [ ] #7148 Discussion Over
- [ ] #7174 Discussion Over
- [ ] #7188 Already Closed
- [ ] #7217 Valid
- [ ] #7227 Already Closed
- [ ] #7230 Unsure - Can't test / no OSX install
- [ ] #7257 Valid
- [ ] #7285 Already Closed

**Comment by alimoeeny** [comment]
[ ] #5492 Can't Reproduce

**Comment by rgrinberg** [comment]
- [ ] #6465 Cannot reproduce
- [ ] #4380 Unsure. Author has proposed an enhancement, but hasn't made an official PR.
- [ ] #4381 Unsure. I was unable to reproduce, but the author didn't leave clear instructions on how to do it
- [ ] #4910 Can't reproduce
- [ ] #4973 Valid
- [ ] #5423 Wontfix. This requires upstream fixes in outshine, evil, smart parens, line numbers, not spacemacs
- [ ] #6376 Valid
- [ ] #6643 Valid. But hard to reproduce
- [ ] #6765 Valid
- [ ] #6870 Valid

**Comment by NJBS** [comment]
- [ ] #4239 Can't reproduce
- [ ] #4242 Can't reproduce using `company-emoji`.
- [ ] #4330 Unsure, spacemacs now warns about killing buffers outside of your current layout so the user can simply answer `q` to prevent projectile from killing the buffers in the other layout. However, this is quite inconvenient when the layouts share many files from the same project.
- [ ] #4657 Fixed, this PR was cherry picked properly but the PR wasn't closed.
- [ ] #4869 Fixed, I believe the latest version of the main site no longer has these issues in the documentation.
- [ ] #4922 Valid, however, conflicts must be resolved and the binding conflicts with the toggle proposed in #4262.
- [ ] #5435 Unsure, it appears to be valid but I cannot try to reproduce as it is an OSX specific issue.
- [ ] #5456 No Response, this should be fixed in the latest release but the original issue author hasn't confirmed it yet (almost 2 months now).
- [ ] #5554 Can't reproduce, doesn't seem there's an easy list of reproduction steps.
- [ ] #5590 Can't reproduce, the timer is properly set when trying to reproduce on the latest `develop`.
- [ ] #5620 Valid, however, conflicts must be resolved.
- [ ] #5654 Valid
- [ ] #5751 Valid, should eventually be fixed by #6090.
- [ ] #6189 Valid
- [ ] #6312 No Response
- [ ] #6701 Unsure, this is an issue with the user's connection to the package repositories, I'm unsure what Spacemacs can do about this.
- [ ] #6893 Valid
- [ ] #7031 Discussion Over, was never reported to `evil`.
- [ ] #7247 Already Closed
- [ ] #7265 Valid, to be reproduced should probably be removed since a couple of users have complained about the same problem.

**Comment by gilbertw1** [comment]
- [ ] #3677 Discussion Over - no discussion on this topic in nearly a year
- [ ] #3717 Can't Reproduce
- [ ] #3811 Valid - Looks valid, but PR has conflicts
- [ ] #3867 No Response - PR has no response or activity in nearly a year and CI checks fail
- [ ] #4075 Fixed - Associated PR was merged
- [ ] #4130 Can't Reproduce - fixed in master
- [ ] #4167 No Response - feature request appears valid
- [ ] #4243 Discussion Over - Valid issue, workaround provided, no activity in 5 months
- [ ] #4365 Discussion Over
- [ ] #4416 Valid - waiting on PR merge
- [ ] #4422 No Response - no activity on PR since Jan, unclear if it would ever be accepted
- [ ] #4453 Valid
- [ ] #4640 Discussion Over
- [ ] #4662 Valid - reproduced in master & develop
- [ ] #5012 No Response - change proposed to PR author, but no response in 7 months
- [ ] #5172 Unsure - no OSX install to reproduce
- [ ] #5292 No Response - PR appears valid, but has received no response in ~7 months
- [ ] #5488 Valid
- [ ] #5529 Unsure - Can't reproduce on linux, no OSX install available
- [ ] #5560 Valid - waiting on PR merge
- [ ] #5820 Discussion Over - Question answered, satisfactory workaround found
- [ ] #5823 No Response
- [ ] #5827 No Response - appears to be a valid issue for colemak users
- [ ] #6030 Valid
- [ ] #6090 Valid - appears valid, fails CI checks + no activity in several months
- [ ] #6348 Discussion Over - original issue was answered
- [ ] #6351 Valid - upstream emacs / org mode issue
- [ ] #6454 Valid
- [ ] #6635 Valid - upstream issue w/ company
- [ ] #6689 Discussion Over - Windows smooth scroll issue / disabled by default now
- [ ] #7098 Already Closed
- [ ] #7130 Discussion Over
- [ ] #7131 Valid
- [ ] #7136 Discussion Over
- [ ] #7161 Valid - emacs issue
- [ ] #7225 Can't Reproduce - however multiple users reported problem
- [ ] #7226 Valid - user question unanswered
- [ ] #7228 Can't Reproduce - I run XMonad on Arch and can't reproduce reported behavior
- [ ] #7318 Can't Reproduce - this looks like an issue with the reporter's setup
- [ ] #7329 Valid

**Comment by smile13241324** [comment]
- [ ] #3755 Discussion Over - User requested the "next error" function to cycle when it reached the last error. However there has not been a PR for this for almost a year now, so I hardly think this is still relevant.
- [ ] #3946 Discussion Over - PR has been made however it contains conflicts and is waiting since 27.11.2015 to be merged. I don't think that this PR is still relevant after all this time.
- [ ] #4154 Discussion Over - PR has been made however it contains conflicts and is unchanged since over a month. Package creator wrote that he ended up installing his package locally and has no interest in working further on the PR. In my opinion this PR is dead and the ticket should be closed.
- [ ] #4155 Valid - PR waiting to be merged.
- [ ] #4246 No Response - New Feature request, no answer since end 2015.
- [ ] #4247 Discussion Over - A new feature was requested however no PR has been created though since end 2015. I think this can be closed.
- [ ] #4708 Unsure - Needs to be tested on OSX.
- [ ] #5280 Discussion Over - PR has been created but has conflicts, last response was on 02.08.2016. As nobody has continued working on this PR I think this one is dead.
- [ ] #5543 Can't Reproduce - Tried to reproduce on develop on Ubuntu 16.04, was able to delete buffers from my clientemacs without issues.
- [ ] #5609 Valid - User complains about no fringe between the line numbers and his code in terminal mode, tested this on arch linux and it is true in terminal there is no margin between the line numbers and the code. However a solution for the .spacemacs file has been provided.
- [ ] #6018 Can't Reproduce - Downgraded my spacemacs to only use the base distribution and checked the helm action selection, worked as expected on develop.
- [ ] #6067 Valid
- [ ] #6104 Can't Reproduce - Started spacemacs as deamon, then started two separate clients as described in the ticket did not encounter any errors. Tested on Ubuntu 16.04.
- [ ] #6269 Can't Reproduce - Tried to create persistent highlight on develop by using `C-SPC` but whenever I press `ESC` the visual state is lost. So it seems that this issue has been fixed upstream or the default behavior of spacemacs has been changed to make the visual state behave consistent throughout the application.
- [ ] #6380 Valid
- [ ] #6468 Unsure - Not sure I understand the problem on Ubuntu 16.04 I don't experienced any issues also tried on arch linux not an issue here either. However someone should try to reproduce that on osx as this was where the issue was reported first.
- [ ] #6587 Discussion Over - New package, has been created in separate issue, this thread seems to be dead. Package issue #6604.
- [ ] #6621 Discussion Over - New package request for a new free collaboration tool. As floobits was not accepted as an answer and no further attempts to clarify which other tool to implement were made the discussion stopped 28 days ago. I think this can be closed.
- [ ] #6631 Valid
- [ ] #6655 Can't Reproduce - Tried to reproduce on Ubuntu 16.04 and failed.
- [ ] #6659 Can’t Reproduce - Tried it on develop on linux. Used one of the official sample latex files found no issues with autocomplete or yasnipet.
- [ ] #6973 Valid
- [ ] #7026 No Response - User complains about wrong cursor colors in various modes, no response to this ticket for over a month.
- [ ] #7041 No Response - New feature request to persist the selected theme without adjusting the .spacemacs file. Has not been answered for over a month.
- [ ] #7186 No Response - New feature request, no response since 26 days.

**Comment by bleggett** [comment]
- [ ] #3977 Valid
- [ ] #4107 No Response
- [ ] #4335 Valid
- [ ] #4362 Wontfix
- [ ] #4612 Valid
- [ ] #4617 No Response
- [ ] #4865 No Response
- [ ] #5483 Fixed
- [ ] #5740 Can't Reproduce
- [ ] #5917 Valid
- [ ] #6344 No Response
- [ ] #6388 Valid
- [ ] #6435 Valid
- [ ] #6650 Can't Reproduce - Emacs25/Linux
- [ ] #6709 Valid
- [ ] #7038 Valid
- [ ] #7062 No Response
- [ ] #7183 Fixed
- [ ] #7232 Wontfix  - not enough info
- [ ] #7259 Valid

**Comment by bmag** [comment]
- [ ] #3691 Can't reproduce. The main culprit `, t a` no longer uses `spacemacs//cider-eval-in-repl-no-focus`. Could be NRA, or Valid but the symptoms/problematic commands have changed. This issue went mostly above my head.
- [ ] #4597 Unsure. It's an FR to change the loading propress, so the user can solve a problem that sounds to me like it could be solved in `user-init` with a simple `with-eval-after-load` or a `spacemacs|use-package-add-hook`. Not sure if the request made sense back then, personally sounds like a Wontfix to me, but it's probably NRA by now anyway.
- [ ] #5123 Valid
- [ ] #5651 Can't reproduce. Not enough info.
- [ ] #6442 Valid
- [ ] #6483 Valid
- [ ] #6568 Valid
- [ ] #6868 Valid. PR pending.
- [ ] #7162 Fixed
- [ ] #7309 Valid

**Comment by NJBS** [comment]
- [ ] #3703 No Response, I could have sworn there was another issue about this exact behavior opened in the past but I'm unable to find it.
- [ ] #3824 Can't Reproduce, I believe the changes to smooth-scrolling (namely removing the external package) have fixed this issue.
- [ ] #5163 No Response
- [ ] #5170 Valid
- [ ] #5666 Can't Reproduce, I can use surround and cleverparens as expected.
- [ ] #6028 Valid
- [ ] #6279 Valid, possibly beginner friendly/ready for work.
- [ ] #6423 Discussion Over
- [ ] #6426 Can't Reproduce
- [ ] #6485 No Response
- [ ] #6651 No Response
- [ ] #6684 Valid
- [ ] #6793 Valid
- [ ] #6924 Unsure, however, I can reproduce this but I'm not sure what Spacemacs can do about it.
- [ ] #7083 Valid
- [ ] #7149 Fixed on develop, #7096, cd6b5ed67b8e
- [ ] #7270 Valid
- [ ] #7271 Valid, reported upstream at https://bitbucket.org/lyro/evil/issues/719/ so the `should report upstream` tag should be changed.

**Comment by lazywithclass** [comment]
- [ ] #3738 Valid
- [ ] #3828 Discussion Over, almost an year over
- [ ] #3906 Discussion Over, even though it seems Valid
- [ ] #4544 Fixed
- [ ] #4803 Discussion Over
- [ ] #4964 Valid
- [ ] #5233 Valid
- [ ] #5658 Valid
- [ ] #5974 Valid
- [ ] #6128 No Response
- [ ] #6135 Valid
- [ ] #6518 Valid
- [ ] #6799 Valid
- [ ] #6923 Already closed
- [ ] #6952 Valid
- [ ] #7069 Valid
- [ ] #7085 Already closed
- [ ] #7124 Already closed
- [ ] #7260 Valid
- [ ] #7323 Valid

**Comment by rgrinberg** [comment]
- [ ] #3934 Can't reproduce with emacs 25
- [ ] #4138 Valid. Waiting for author to respond
- [ ] #4146 NRA. Font problem
- [ ] #4351 Valid. Although could also be upstream emacs bug
- [ ] #4957 Fixed
- [ ] #5443 Wontfix. (Already closed)
- [ ] #5467 Can't reproduce.
- [ ] #5599 Can't reproduce.
- [ ] #6008 NRA. Needs to be reported upstream.
- [ ] #6872 No Response.

**Comment by ekmecic** [comment]
- [ ] #4023 No response
- [ ] #4816 No response
- [ ] #5481 No response
- [ ] #5684 Valid
- [ ] #5715 Valid
- [ ] #6611 Valid
- [ ] #6995 Fixed in Develop
- [ ] #7185 Valid
- [ ] #7218 Valid
- [ ] #7240 Fixed in Develop

**Comment by inirudebwoy** [comment]
- [ ] #5681 Fixed
- [ ] #5957 Can't reproduce
- [ ] #7032 Already closed
- [ ] #7239 Already closed
- [ ] #7311 Valid

**Comment by jredville** [comment]
- [ ] #6978 Valid
- [ ] #3651 Question, Discussion over
- [ ] #3729 NRA (upstream bug)
- [ ] #4323 Discussion over
- [ ] #4507 Not verified, don't have windows to verify. No activity since May
- [ ] #4884 Possibly Valid, no response since Jan
- [ ] #5552 Fixed
- [ ] #5737 Valid, PR pending
- [ ] #6307 Valid, possible rebase needed
- [ ] #7019 Fixed, I think
- [ ] #7187 Valid

**Comment by kdelwat** [comment]
- [ ] #3864 Valid
- [ ] #4177 Valid
- [ ] #4595 Wontfix (upstream issue)
- [ ] #5126 Valid
- [ ] #5447 Valid
- [ ] #5514 Valid
- [ ] #5570 No Response (lack of information)
- [ ] #5577 Valid, No Response
- [ ] #5778 Valid
- [ ] #5802 Wontfix
- [ ] #5906 Valid
- [ ] #6177 Can't reproduce (waiting for response)
- [ ] #6402 Valid
- [ ] #6573 Already closed
- [ ] #6665 Discussion Over (although I don't think the layer was ever added)
- [ ] #6931 No response
- [ ] #7014 Already closed
- [ ] #7178 Valid
- [ ] #7201 Already closed
- [ ] #7241 Already closed

**Comment by mahinshaw** [comment]
- [ ] #3859 Valid - However build is failing, and therefore the patch is out of date.
- [ ] #3972 No Response
- [ ] #3995 Valid
- [ ] #3997 No Response
- [ ] #4049 Valid
- [ ] #4128 Valid
- [ ] #4425 Valid
- [ ] #5018 No Response
- [ ] #5288 Already Closed
- [ ] #6044 Valid
- [ ] #6082 Valid
- [ ] #6120 Can't Reproduce
- [ ] #6339 Valid
- [ ] #6399 No Response - From the conversation, it sounds like this is Fixed.
- [ ] #6420 Already Closed
- [ ] #6467 Valid
- [ ] #6539 Valid
- [ ] #7012 Valid
- [ ] #7096 Already Closed
- [ ] #7251 Valid

**Comment by gilbertw1** [comment]
- [ ] #3960 Valid - upstream bug
- [ ] #4112 Valid
- [ ] #4564 Valid
- [ ] #5439 Valid - Should be closed as duplicate of #3590
- [ ] #5869 Discussion Over - OSX specific issue w/ sufficient workaround provided
- [ ] #6057 Valid
- [ ] #6989 Valid - PR Pending
- [ ] #7078 Valid - emacs bug
- [ ] #7128 Discussion Over - non issue, malformed emacs package header on a github project
- [ ] #7143 Valid - upstream issue

**Comment by d12frosted** [comment]
Thanks everyone for participating in this event! Bit by bit I validate all your input and I see that this event helped to revive some old and really valid issues / PRs and to amputate oldies that no one cares about. And most important - I see some fresh faces here, which is great! 

Thanks for help! 😸
